### DISCUSSION TITLE
Additional Handling Edge Cases

### DISCUSSION THREAD

**Discussion by ZiyangWang001007081** [ZiyangWang001007081/Tafe230243#5] [General]
Hi Team,

As I'm writing the C# code for currency conversion, I'm thinking about handling edge cases, such as invalid input or unexpected issues. I'm thinking of including validation for empty values, also for numeric values. 

What other error handling and messaging strategies should we implement to provide a smooth user experience and  handle any unexpected situations? Or will these 2 cases be sufficient?

Please let me know what do you think. Thank you.

Cheers,
Ziyang Wang

**Comment by Abhishek-Sathe-Tafe** [comment]
I think let us keep it simple. We will just validate the empty/null input. Thanks!
